### PR TITLE
GSmartControl.GSmartControl version 1.1.3

### PR DESCRIPTION
**Pull request by OyePradumya**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/29136)

### CODE REVIEW

File: manifests/g/GSmartControl/GSmartControl/1.1.3/GSmartControl.GSmartControl.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: GSmartControl.GSmartControl
+PackageVersion: 1.1.3
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://nchc.dl.sourceforge.net/project/gsmartcontrol/1.1.3/gsmartcontrol-1.1.3-win64.exe
```
Comment (ItzLevvie) [inline manifests/g/GSmartControl/GSmartControl/1.1.3/GSmartControl.GSmartControl.installer.yaml:8]:
```suggestion
  InstallerUrl: https://sourceforge.net/projects/gsmartcontrol/files/1.1.3/gsmartcontrol-1.1.3-win64.exe/download
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/g/GSmartControl/GSmartControl/1.1.3/GSmartControl.GSmartControl.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: GSmartControl.GSmartControl
+PackageVersion: 1.1.3
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://nchc.dl.sourceforge.net/project/gsmartcontrol/1.1.3/gsmartcontrol-1.1.3-win64.exe
+  InstallerSha256: F33064B67B7399BEA4D106862AFE3A6F36EDF7200A7B6B1BE68A39FED4E39E6D
```
Comment (ItzLevvie) [inline manifests/g/GSmartControl/GSmartControl/1.1.3/GSmartControl.GSmartControl.installer.yaml:9]:
```suggestion
  InstallerSha256: F33064B67B7399BEA4D106862AFE3A6F36EDF7200A7B6B1BE68A39FED4E39E6D
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
missing https://github.com/microsoft/winget-pkgs/pull/29136/files#r720103526

**Review by ItzLevvie — DISMISSED** [review]

File: manifests/g/GSmartControl/GSmartControl/1.1.3/GSmartControl.GSmartControl.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+PackageIdentifier: GSmartControl.GSmartControl
+PackageVersion: 1.1.3
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://sourceforge.net/projects/gsmartcontrol/files/1.1.3/gsmartcontrol-1.1.3-win64.exe/download
+  InstallerSha256: F33064B67B7399BEA4D106862AFE3A6F36EDF7200A7B6B1BE68A39FED4E39E6D
+  Scope: machine
+  Scope: machine
+  Scope: machine
```
Comment (ItzLevvie) [inline manifests/g/GSmartControl/GSmartControl/1.1.3/GSmartControl.GSmartControl.installer.yaml:12]:
```suggestion
```
You added this two times.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by OyePradumya** [comment]
You just suggest whatever you want. I will commit every damn suggestion.. but let it merged. @ItzLevvie

**Review by ItzLevvie — APPROVED** [review]

**Comment by OyePradumya** [comment]
@msftbot Merge this.

**Comment by ItzLevvie** [comment]
> @msftbot Merge this.

You will need to wait for an administrator to take a look at this PR.

**Pull request merged**